odxtools/cli/browse.py

```diff
@@ -4,7 +4,7 @@
 import sys
 from typing import List, Optional, Union, cast
 
-import InquirerPy.prompt as PI_prompt
+import InquirerPy.prompt as IP_prompt
```

Comment:
What? are you sure?

Reply:
the history of this is that in its current state `pyright`, complains about this:
```
$ pyright
/home/and/src/odxtools/odxtools/cli/browse.py
  /home/and/src/odxtools/odxtools/cli/browse.py:7:8 - error: Import "InquirerPy.prompt" could not be resolved (reportMissingImports)
  /home/and/src/odxtools/odxtools/cli/browse.py:100:14 - error: Module is not callable (reportCallIssue)
[...]
```

while if this issue is fixed by replacing the line with `from InquirerPy import prompt as IP_prompt  # type: ignore[attr-defined]`, mypy complains about the `# type: ignore` comment being unnecessary (at least on the GH actions runner, locally it is fine on my machine). the `PI` to `IP` rename is a residual of that, because I thought that if I have to change that line anyway, I could use a name that makes sense...